src/openrct2/config/Config.cpp

```diff
@@ -212,7 +211,7 @@ namespace OpenRCT2::Config
 
             // Default config setting is false until the games canvas can be separated from the effect
             model->DayNightCycle = reader->GetBoolean("day_night_cycle", false);
-            const bool isHardware = model->DrawingEngine != DrawingEngine::Software;
+            const bool isHardware = true;
             model->EnableLightFx = isHardware && reader->GetBoolean("enable_light_fx", false);
             model->EnableLightFxForVehicles = isHardware && reader->GetBoolean("enable_light_fx_for_vehicles", false);
```

Comment:
I realise this used to say `!= DrawingEngine::Software`, but lightfx have never worked on OpenGL, right? I think it would therefore be better to check against OpenGL instead, i.e. only save these settings when the software renderer is used.

Reply:
True, I didn't look at the variable only the condition.